framework/branch-key-store.md

```diff
@@ -522,7 +522,7 @@ List of TransactWriteItem:
     - “create-time” (S): `timestamp`
     - "kms-arn" (S): configured KMS Key
     - “hierarchy-version” (N): either `1` or `2`, depending on the `hierarchy-version`
-    - Every key-value pair of the custom [encryption context](./structures.md#encryption-context-3) that is associated with the branch key
+    - Every key-value pair in the [encryption context](./structures.md#encryption-context) provided by users and is associated with the branch key
```

Comment:
```suggestion
    - Every key-value pair in the `encryption-context` and is associated with the branch key
```

Reply:
I think you want to reference the prefix. I would way something like all the additional pairs of the branch key context.